index.js

```diff
@@ -1,118 +1,235 @@
+if (dogFur != "Fluffy Black & white"){
+    dog = "Defintely not Sofie";
+}
+console.log (dog)
 
 
 //Task c: Convert string ("1999") to integer (1999)  (no function required) // hint look up the Number method
 
+console.log(parseInt("1999"));
 
 
 
 
-//Task d: Write a function to multiply a*b 
-
-
+//Task d: Write a function to multiply a*b
 
+function multiply(a, b){
+    return a*b;
+}
+console.log(multiply(10, 20))
 
 
 /************************************************************** Task 2 **************************************************************/
 //Age in Dog years
-//write a function that takes your age and returns it to you in dog years - they say that 1 human year is equal to seven dog years 
-
-
-
+//write a function that takes your age and returns it to you in dog years - they say that 1 human year is equal to seven dog years
 
+let ageInDogYears;
+let humanAge;
+let ageConverter = function(humanAge) {
+    ageInDogYears = humanAge * 7;
+    console.log("I am " + ageInDogYears + " in dog years.");
+    }
+    ageConverter(32);
+// returning undefined for some reason... grrr
 
 /************************************************************** Task 3 **************************************************************/
-//Dog feeder 
+//Dog feeder
 //takes weight in pounds and age in years (note if the dog is a puppy the age will be a decimal) and returns the number of pounds of raw food to feed in a day.
 
 //feeding requirements
-// adult dogs at least 1 year 
+// adult dogs at least 1 year
 // up to 5 lbs - 5% of their body weight
-// 6 - 10 lbs - 4% of their body weight 
-// 11 - 15 lbs - 3% of their body weight 
-// > 15lbs - 2% of their body weight 
+// 6 - 10 lbs - 4% of their body weight
+// 11 - 15 lbs - 3% of their body weight
+// > 15lbs - 2% of their body weight
 
 // Puppies less than 1 year
 // 2 - 4 months 10% of their body weight
-// 4 - 7 months 5% of their body weight 
+// 4 - 7 months 5% of their body weight
 // 7 - 12 months 4% of their body weight
 
 // when you are finished invoke your function with the weight of 15 lbs and the age of 1 year - if your calculations are correct your result should be 0.44999999999999996
-  
-
 
 
+let dogFeeder = function (age, weight){
+    if( age >= 1 ) {
+        if( weight <= 5 ) {
+            feed = weight * .05;
+        } else if ( 5 < weight && weight <= 10 ) {
+            feed = weight * .04;
+        } else if ( 10 < weight && weight <= 15 ) {
+            feed = weight * .03;
+        } else if ( weight < 15 ) {
+            feed = weight * .02;
+        // } else {
+        //     console.log("Please try again with dog's weight")
+        }
+    } else if ( (2/12) < age && age < (4/12) ) {
+        feed = weight * .1;
+    } else if ( (4/12) <= age && age < (7/12) ) {
+        feed = weight * .05;
+    } else if ( (7/12) <= age && age < 1 ) {
+        feed = weight * .04;
+    }
+    console.log("Feed Me " + feed + " pounds of raw food a day");
+}
+dogFeeder(1, 15);
```

Comment:
Other than the 1 error, this works. It could be easier to understand though, and there's some extraneous logic.